packages/app/src/app/overmind/namespaces/editor/actions.ts

```diff
@@ -239,6 +239,12 @@ export const sandboxChanged: AsyncAction<{ id: string }> = withLoadApp<{
     state.editor.modulesByPath = fs;
   });
 
+  if (sandbox.featureFlags?.containerLsp) {
```

Comment:
We start or stop based on the experiment?

Reply:
Yes, this is only related to containers running the LSP experiment, it is not an issue when LSP is in client